pkg/util/mount/unmount.go

```diff
@@ -18,17 +18,18 @@ package mount
 
 import (
 	"fmt"
-	"log"
 	"os"
+
+	"go.uber.org/zap"
 )
 
 // UnmountPath is a common unmount routine that unmounts the given path and
 // deletes the remaining directory if successful.
-func UnmountPath(mountPath string, mounter Interface) error {
+func UnmountPath(logger *zap.SugaredLogger, mountPath string, mounter Interface) error {
 	if pathExists, pathErr := PathExists(mountPath); pathErr != nil {
 		return fmt.Errorf("Error checking if path exists: %v", pathErr)
 	} else if !pathExists {
-		log.Printf("Warning: Unmount skipped because path does not exist: %v", mountPath)
+		logger.With("mount path", mountPath).Warn("Unmount skipped because path does not exist.")
```

Comment:
Rather than doing `With("mountPath", mountPath)` multiple times (note no spaces in field names) I'd instead reassign logger:

```go
logger := logger.With("mountPath", mountPath)